ray-operator/controllers/ray/utils/util_test.go

```diff
@@ -530,50 +533,133 @@ func TestGetWorkerGroupDesiredReplicas(t *testing.T) {
 	workerGroupSpec.MaxReplicas = &minReplicas
 	workerGroupSpec.Suspend = &suspend
 	assert.Zero(t, GetWorkerGroupDesiredReplicas(ctx, workerGroupSpec))
+
+	// Test 7: `WorkerGroupSpec.NumOfHosts` is 3.
```

Comment:
I haven't seen any real accelerators with `NumOfHosts` equal to 3. Perhaps we should use 4 instead to be closer to real-world accelerators.